_includes/exhibits.js

```diff
@@ -0,0 +1,1929 @@
+
+      return false;
+    });
+
+    this.viewer.addHandler('canvas-enter', function(e) {
+      const THIS = e.userData;
+      THIS.faster();
+    }, this);
+
+    this.viewer.addHandler('canvas-drag', function(e) {
+      const THIS = e.userData;
+      const overlay = $('#' + THIS.currentOverlay);
+      const position = THIS.normalize(e.position);
+      if (THIS.drawing == 1) {
+        THIS.drawing = 2;
+        e.preventDefaultAction = true;
+        THIS.drawLowerBounds(position);
+      }
+      else if (THIS.drawing == 2) {
+        e.preventDefaultAction = true;
+        THIS.drawUpperBounds(position);
+      }
+    }, this);
+
+    this.viewer.addHandler('canvas-drag-end', function(e) {
+      const THIS = e.userData;
+      const position = THIS.normalize(e.position);
+      if (THIS.drawing == 2) {
+        e.preventDefaultAction = true;
+        THIS.finishDrawing(position);
+        THIS.pushState();
+      }
+    }, this);
+
+    this.viewer.addHandler('canvas-click', function(e) {
+      const THIS = e.userData;
+      const overlay = $('#' + THIS.currentOverlay);
+      const position = THIS.normalize(e.position);
+      if (THIS.drawing == 1) {
+        THIS.drawing = 2;
+        e.preventDefaultAction = true;
+        THIS.drawLowerBounds(position);
+      }
+      else if (THIS.drawing == 2) {
+        e.preventDefaultAction = true;
+        THIS.finishDrawing(position);
+        THIS.pushState();
+      }
+    }, this);
+
+    $(this.viewer.element).mousemove(this, function(e) {
+      const THIS = e.data;
+      THIS.mouseXY = e;
+      if (THIS.drawing == 2) {
+        THIS.drawUpperBounds(THIS.mouseXY);
+      }
+    });
+
+    this.viewer.addHandler('animation', function(e) {
+      const THIS = e.userData;
+      const scale = THIS.viewer.viewport.getZoom();
+      const pan = THIS.viewer.viewport.getCenter();
+      THIS.v = [
+        round4(scale),
+        round4(pan.x),
+        round4(pan.y)
+      ];
+      THIS.newView(false); 
+    }, this);
+
+    this.viewer.addHandler('animation-finish', function(e) {
+      const THIS = e.userData;
+      const scale = THIS.viewer.viewport.getZoom();
+      const pan = THIS.viewer.viewport.getCenter();
+      THIS.v = [
+        round4(scale),
+        round4(pan.x),
+        round4(pan.y)
+      ];
+      THIS.pushState();
+      THIS.faster();
+    }, this);
+
+    // Display viewer
+    this.finishAnimation();
+    this.viewer.setVisible(true);
+  },
+
+  /*
+   * Editor buffers
+   */ 
+
+  get bufferWaypoint() {
+    if (this.state.buffer.waypoint === undefined) {
+      const viewport = this.viewport;
+      const group = this.group;
+      return {
+        Zoom: viewport.scale,
+        Pan: [
+          viewport.pan.x,
+          viewport.pan.y
+        ],
+        Group: group.Name,
+        Description: '',
+        Name: 'Untitled',
+        Overlay: {
+          x: -100,
+          y: -100,
+          width: 200,
+          height: 200,
+        },
```

Comment:
This overlay is well outside the normalized ranged of [0,1]. It mimics the appearance of having no overlay by simply drawing the overlay off the screen.